packages/svelte/tests/migrate/samples/event-handlers/output.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+	let { onclick, ontoggle, 'custom-event-bubble': oncustom_event_bubble } = $props();
+</script>
+
+<button  onclick={(payload) => {
+	console.log('hi')
+
+	onclick?.(payload);
+}}>click me</button>
+<button   onclick={(payload_1) => {
```

Comment:
parameters don't need to be globally unique, they just need to deconflict with whatever's declared or referenced in the function body